lib/teracy-dev-essential/config/host_manager.rb

```diff
@@ -37,10 +43,43 @@ def configure_node(settings, config)
           provision.set_options(options)
         end
 
+        @plugin_aliases.unshift hostname
+
+        check_conflict_hostname @plugin_aliases
       end
 
       private
 
+      def check_conflict_hostname aliases
+        @logger.debug('check conflict hostname')
+
+        etc_hosts = `cat /etc/hosts`
```

Comment:
this path is different between OSes, especially Windows, make sure this is portable.